Dockerfile

```diff
@@ -74,3 +79,47 @@ RUN	ln -s /opt/local/lib/livepeer-w3/livepeer-w3.js /usr/local/bin/livepeer-w3 &
 EXPOSE	1935	4242	8080	8889/udp
 
 CMD	["/usr/local/bin/MistController", "-c", "/etc/livepeer/catalyst.json"]
+
+FROM catalyst AS livepeer-in-a-box
+
+ARG TARGETARCH
+
+RUN	apt update && apt install -yqq \
+	rabbitmq-server \
+	nginx \
+	gdb \
+	inotify-tools \
+	file \
+	# for `shasum`
```

Comment:
go-lang has built-in sha256sum https://pkg.go.dev/crypto/sha256#New 
installing `shasum` is fine... easier solution for now.

Reply:
Yeah, we really just need catalyst-downloader to make use of that, will open a ticket